langchain/src/document_loaders/tests/github.int.test.ts

```diff
@@ -0,0 +1,11 @@
+import { test } from "@jest/globals";
+import { GithubRepoLoader } from "../github.js";
+
+test("Test GithubRepoLoader", async () => {
+  const loader = new GithubRepoLoader(
+    "https://github.com/hwchase17/langchainjs",
+    { branch: "main", recursive: false, unknown: "warn" }
+  );
+  const documents = await loader.load();
+  console.log(documents[0].pageContent);
+}, 10000);
```

Comment:
Btw we don't need this timeout on each test, I've set it here for all integration tests https://github.com/hwchase17/langchainjs/blob/810d8be60903ba6e2a7941811b179a8fbd1a3112/langchain/package.json#L64